engine/src/main/resources/assets/skins/mainmenu.skin

```diff
@@ -3,7 +3,7 @@
     "elements": {
         "UIScreenLayer": {
             "background-scale-mode": "scale_fill",
-            "background": "menuBackground"
+            "background": "menuBackgroundJPEG"
```

Comment:
I suggest removing the PNG - having the same image twice doesn't buy you anything.
Also, please convert all background images.